Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/controller/DrawerController.java

```diff
@@ -62,14 +68,20 @@
 public class DrawerController
         extends Controller
         implements DrawerAdapter.Callback, View.OnClickListener {
+    private static final String TAG = "DrawerController";
+
     protected FrameLayout container;
     protected DrawerLayout drawerLayout;
     protected LinearLayout drawer;
     protected RecyclerView recyclerView;
     protected DrawerAdapter drawerAdapter;
+    @Nullable
+    private CompositeDisposable compositeDisposable = null;
```

Comment:
Probably shouldn't make it nullable. Just create it once and then use clear() instead of dispose()